doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -50,36 +50,40 @@ int main(int argc, char **argv)
   ros::AsyncSpinner spinner(1);
   spinner.start();
 
-
   /* This sleep is ONLY to allow Rviz to come up */
   sleep(20.0);
 
   // BEGIN_TUTORIAL
   //
   // Setup
   // ^^^^^
-  //
+
+  // MoveIt! operates on sets of joints called "planning groups" and stores them in an object called
+  // the `JointModelGroup`. Throughout MoveIt! the terms "planning group" and "joint model group"
+  // are used interchangably.
+  static const std::string PLANNING_GROUP = "right_arm";
+
   // The :move_group_interface:`MoveGroup` class can be easily
-  // setup using just the name
-  // of the group you would like to control and plan for.
-  moveit::planning_interface::MoveGroupInterface group("right_arm");
+  // setup using just the name of the planning group you would like to control and plan for.
+  moveit::planning_interface::MoveGroupInterface move_group(PLANNING_GROUP);
 
-  // We will use the :planning_scene_interface:`PlanningSceneInterface`
-  // class to deal directly with the world.
-  moveit::planning_interface::PlanningSceneInterface planning_scene_interface;
+  // Raw pointers are frequently used to refer to the planning group for improved performance.
+  const robot_state::JointModelGroup *joint_model_group =
+      move_group.getCurrentState()->getJointModelGroup(PLANNING_GROUP);
 
   // (Optional) Create a publisher for visualizing plans in Rviz.
-  ros::Publisher display_publisher = node_handle.advertise<moveit_msgs::DisplayTrajectory>("/move_group/display_planned_path", 1, true);
+  ros::Publisher display_publisher =
+      node_handle.advertise<moveit_msgs::DisplayTrajectory>("/move_group/display_planned_path", 1, true);
   moveit_msgs::DisplayTrajectory display_trajectory;
 
   // Getting Basic Information
   // ^^^^^^^^^^^^^^^^^^^^^^^^^
   //
   // We can print the name of the reference frame for this robot.
-  ROS_INFO("Reference frame: %s", group.getPlanningFrame().c_str());
+  ROS_INFO("Reference frame: %s", move_group.getPlanningFrame().c_str());
 
   // We can also print the name of the end-effector link for this group.
-  ROS_INFO("Reference frame: %s", group.getEndEffectorLink().c_str());
+  ROS_INFO("Reference frame: %s", move_group.getEndEffectorLink().c_str());
```

Comment:
I suppose, this should actually be "Endeffector" instead of "Reference frame". (I know this is no mistake in your patch...)